polls/templatetags/poll_votings.py

```diff
@@ -0,0 +1,24 @@
+
+from copy import copy
+
+from django import template
+
+from polls.models import Question
+
+register = template.Library()
+
+
+@register.inclusion_tag('polls/poll_page.html',
+                        takes_context=True)
+def poll_page(context, pk=None, page=None):
+    context = copy(context)
+    polls = Question.objects.filter(page=page.id)
+    if polls:
+        context.update({
+            'questions': polls
+        })
+    else:
+        context.update({
+            'error': True,
+        })
```

Comment:
Why do we need to return an `error` here? Why not an empty `[]` for the `questions` key? If a `Page` doesn't have any set questions then we can handle that in the template without needing a separate context variable set, I think that only makes the template more complex than necessary.